bin/deploy.sh

```diff
@@ -25,18 +25,26 @@ export DRONE_SOURCE_BRANCH=$(echo $DRONE_SOURCE_BRANCH | tr '[:upper:]' '[:lower
 
 if [[ ${KUBE_NAMESPACE} == ${BRANCH_ENV} ]]; then
   $kd -f kube/configmaps -f kube/certs
-  $kd -f kube/redis -f kube/app -f kube/file-vault
+  $kd -f kube/redis -f kube/file-vault -f kube/app
 elif [[ ${KUBE_NAMESPACE} == ${UAT_ENV} ]]; then
   $kd -f kube/configmaps/configmap.yml
-  $kd -f kube/redis -f kube/app -f kube/file-vault
+  $kd -f kube/redis
+  $kd -f kube/file-vault/file-vault-ingress.yml -f kube/file-vault/file-vault-service.yml
```

Comment:
by the best practice services suppose to be before ingresses
ingress resources should be applied after deployments and services